### ISSUE TITLE
[feature request] settings option to disable minimized video 

### ISSUE THREAD

**Issue by AwesomelyHumble** [TeamNewPipe/NewPipe#5590]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### How will you/everyone benefit from this feature?

I Iike the fairly new addition of the minimize feature but I find a lot of times I back out of a video but do not want it to continue playing. Can you add a setting to toggle this feature on/off? It's essentially the feature that minimizes and keeps a video when you hit the back button. Thanks!

**Comment by JCU0** [comment]
I agree, it's a very annoying feature because it pops up for every video. With "back" button. With a click on channel icon from a video info screen. Every time. Basically, I am watching non-musical videos, so minimized and background players aren't useful for me. I know, two fingers gesture is a thing but it is not ideal solution to close every video with two fingers.

**What I suggest** is the setting to invert this logic: two fingers gesture should start background player and everything else (back button, channel icon, etc) should just stop&close a video.

P.S. For official YT site and app, an automatic minimized player is the very annoying anti-feature too. It feels like you are loosing control over your devices because they are trying to play when you want them to stop.

**Comment by mhmdanas** [comment]
@AwesomelyHumble if I understand your explanation correctly, you can disable the popup player being opened when leaving NewPipe by changing "Settings > Video and audio > Minimize on app switch" to None. I'll be closing this issue, but if I misunderstood your explanation please make a comment and I'll re-open the issue.

**Comment by AwesomelyHumble** [comment]
@mhmdanas Thank you for looking into this issue. I believe you have it misunderstood as my settings currently reflect your suggestion. What I'm referring to is when a video is playing and you press the back button to return to the previous window (i.e. search results), and the playing video is continuing to play in a smaller window at the bottom of the screen (see attached photo). I don't know what this specific feature is called but it was added a few revisions ago, fairly recently. What I'm requesting is there be an option in the settings menu toggle this from happening. Sometimes I want the video to continue playing while I browse (for example music), but many times I do not. Thank you. 

![Screenshot_20210215-111758~2](https://user-images.githubusercontent.com/28993697/107986003-0a0c3000-6f80-11eb-81ef-0099feb70b24.jpg)

**Comment by AwesomelyHumble** [comment]
Just to add, the feature I'm referring to was added in release version 0.20.0 and called "minimized player" when described in the version release blog post. https://newpipe.net/blog/pinned/release/newpipe-0.20.0/

**Comment by avently** [comment]
see https://github.com/TeamNewPipe/NewPipe/pull/6895. It have options that do something you want
